ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -36,6 +36,15 @@ final class Metrics(val registry: MetricRegistry) {
         registry.meter(Prefix :+ "delayed_submissions")
       val validSubmissions: Meter =
         registry.meter(Prefix :+ "valid_submissions")
+
+      def inputBufferSaturation(party: String): Counter =
```

Comment:
Does it make sense to have these per-party?

Reply:
I think for now this is reasonable, though I'd appreciate more opinions on this. Do we run into problems with 10k unique parties?